test/unittests/execute_floating_point_test.cpp

```diff
@@ -363,6 +385,27 @@ TYPED_TEST(execute_floating_point_types, add)
     EXPECT_THAT(exec(TypeParam{0x0.287p2}, TypeParam{0x1.FFp4}), Result(TypeParam{0x1.048Ep5}));
 }
 
+TYPED_TEST(execute_floating_point_types, abs)
```

Comment:
Put the test by the opcode value, i.e. before `add`.

Reply:
Moved.